bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -196,6 +196,49 @@ public void setBypassTo(int day, int month)
 * *month:* The month until when the bypass should be active
 
 
+```
+public Map<String, Object> getErrorMessages()
```

Comment:
Why is this `Object`?

Reply:
I was following the documentation on https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing - it says this is what should be returned. I assumed this is a prerequisite to make the actions available in the UI as well.
If it's allowed to use a more concrete datatype instead of `Object` I'd be happy to change this.

Reply:
If I see correctly this applies only if you have multiple return values.